### ISSUE TITLE
remove the amd64 monav-server binary from Nemo build

### ISSUE THREAD

**Issue by M4rtinK** [M4rtinK/modrana-packaging#4]
The full modRana tarball contains both arm7 & amd64 compiled monav-server binary to enable easy offline routing without the need for manually obtaining the not-yet-packaged monav-server binary. But the amd64 binary is detected during the build process and creates a dependency on amd64 compiled qt libraries.

As a result, users have to confirm to install the modRana package even though the amd64 dependence can't be satisfied on arm devices.

Basically, just not including the amd64 monav-server binary should fix this.
